docs/articles/baseplatform/index.trst

```diff
@@ -9,6 +9,7 @@ Base platform
     repo-manifests.rst
     reproducible-yocto-builds
     writing-image.rst
+    booting-qemu-image.rst
```

Comment:
Since qemu support is experimental, I would prefer this not to be included in the index. add :orphan: to the top of the article to suppress warnings about it not being included.

Reply:
What about having a complete subsection marked as "experimental" and state in some intro there what that means? Then we could have the convenience of the docs for the users without introducing any expectations on the status.

Reply:
That would make sense, yes.

Reply:
I've updated the branch. I merged both advices: the page is now an orphan, but I've added an "experimental" tag.